qemu/tests/cfg/nested_test.cfg

```diff
@@ -16,3 +16,4 @@
         - boot_l2:
             case_name = boot
             install_node = yes
+        - check_cpu_model_l2:
```

Comment:
Why a empty variants here. It's not a good idea.

Reply:
Case_name is passed by parameter to kar cmd, is there anything required here ?

Reply:
First, a empty variants is acceptable, means no special settings except the main part.
However, i do not think use if `case_name` exist to distinguish the nested test case is a good idea, what if there is new case for nested in the future ?
One possible method id to assign a name for the `variants`, @qcheng-redhat @PaulYuuu

Reply:
@yanan-fu I know we can use an empty variants, but for this test case, it has specific steps like:
```python
            else:
                for i in host_cpu_models:
                    case_name = "x86_cpu_model." + i
                    cpu_model_case_names = cpu_model_case_names + case_name + ","
```
So it's not clear in cfg what this variant wants to do.

Reply:
@PaulYuuu  Yes, it use if `case_name` exist to distinguish the two test case, it not a good idea as i said before. Suggest to use variants name.

Reply:
May be define the `prefix_case_name = x86_cpu_model`. Then in the script, if the variants name is `check_cpu_model_l2`, generate the final case_name
```
for i in host_cpu_models:
    case_name += '%s.%s,' %(prefix_case_name, i)
```

Reply:
Fixed.

Reply:
@PaulYuuu  i am ok with the current change for this conversation now, How about you ?  We can close this conversation if it is ok for you.

Reply:
My suggestion is not to add an extra variable `prefix_case_name`. `case_name` should be a general variable for all nested test.
But indeed, we need to filter those not supported test cases for `check_cpu_model_l2`, so reformatting the `case_name` is necessary:
```python
case_name = ','.join(["%s.%s" % (case_name, _) for _ in host_cpu_models])

Reply:
In this way, the `case_name` defined in the cfg is not the real case name. Reformat in the script.
But i am ok with that too, both are ok for me ~

Reply:
Changed prefix_case_name to case_name.

Reply:
ack